### PR TITLE
feat: autoload helpers by Composer

### PR DESCRIPTION
**Pull request by kenjis**

### CODE REVIEW

File: docs/authentication.md [hunk 1]
```diff
@@ -65,6 +63,10 @@ auth()->id();
 user_id();
 ```
 
+> **Note**
+> The `auth_helper` is autoloaded by Composer. If you want to *override* the functions,
+> you need to define them in `app/Common.php`.
```
Comment (MGatner) [inline docs/authentication.md:68]: I was wrong, Common.php does come first! That is good.

**Review by MGatner — APPROVED** [review]

File: docs/authentication.md (hunk 1 above)
Comment (datamweb) [reply docs/authentication.md:68]: > you need to define them in `app/Common.php`

This was unclear to me too!

**Comment by datamweb** [comment]
@kenjis، in this PR, something else is going to be done?
If not please merge.

**Comment by kenjis** [comment]
Nothing. But only MGater approved.

**Review by datamweb — CHANGES_REQUESTED** [review]

**Comment by datamweb** [comment]
I didn't like it!! Why repeat twice!

Screenshot 2022-08-14 114856

**Comment by kenjis** [comment]
Because it checks the file twice.
`$this->helpers = array_merge($this->helpers, ['auth', 'setting']);` and
`$this->helpers = array_merge($this->helpers, ['setting']);`

It is not easy to show the message once.

File: docs/install.md [hunk 2]
```diff
@@ -117,12 +117,12 @@ class Auth extends ShieldAuth
 }
 ```
 
-2. **Helper Setup** The `auth` and `setting` helpers need to be included in almost every page. The simplest way to do this is to add them to the `BaseController::initController` method:
+2. **Helper Setup** The `setting` helpers need to be included in almost every page. The simplest way to do this is to add them to the `BaseController::initController` method:
```
[the remark this answers is not in the document]
Comment (kenjis) [reply docs/install.md]: Thank you. I've fixed!

**Review by datamweb — APPROVED** [review]

**Comment by kenjis** [comment]
@datamweb I could fix the repeated Skip messages. See https://github.com/codeigniter4/shield/pull/371

**Pull request merged**